src/features/pass-survey/ui/ActivityStepper.tsx

```diff
@@ -116,6 +116,12 @@ function ActivityStepper({
     if (!isForced) {
       trackUserAction(userActionCreator.next());
     }
+
+    const currentStepAnswer = activityStorageRecord?.answers[currentStep];
+
+    if (!currentStepAnswer) {
+      trackUserAction(userActionCreator.skip());
+    }
```

Comment:
I would also move 120-124 to the "else" block. That's in the case when the user made some change to an activity item but he then was forcibly moved to the next item due to the timer